tools/util.py

```diff
@@ -186,6 +186,14 @@ def find_exts(directories, extensions, skip=None):
     return matches
 
 
+def find_exts2(dirs, exts):
```

Comment:
This needs a rebase after the tools clean-up that just landed.
I think it'll make life easier for you; it is much more likely to do the right thing already.

Reply:
rebased